edns.go

```diff
@@ -673,3 +676,94 @@ func (e *EDNS0_PADDING) copy() EDNS0 {
 	copy(b, e.Padding)
 	return &EDNS0_PADDING{b}
 }
+
+const (
+	EDECodeOther uint16 = iota
+	EDECodeUnsupportedDNSKEYAlgorithm
+	EDECodeUnsupportedDSDigestType
+	EDECodeStaleAnswer
+	EDECodeForgedAnswer
+	EDECodeDNSSECIndeterminate
+	EDECodeDNSBogus
+	EDECodeSignatureExpired
+	EDECodeSignatureNotYetValid
+	EDECodeDNSKEYMissing
+	EDECodeRRSIGsMissing
+	EDECodeNoZoneKeyBitSet
+	EDECodeNSECMissing
+	EDECodeCachedError
+	EDECodeNotReady
+	EDECodeBlocked
+	EDECodeCensored
+	EDECodeFiltered
+	EDECodeProhibited
+	EDECodeStaleNXDOMAINAnswer
+	EDECodeNotAuthoritative
+	EDECodeNotSupported
+	EDECodeNoReachableAuthority
+	EDECodeNetworkError
+	EDECodeInvalidData
+)
+
+var EDECodeToString = map[uint16]string{
```

Comment:
might need the reverse as well, just like normal rcodes

Reply:
Seeing as we're not dealing with Rcodes here, is there a good use-case for the reverse lookup?